KIPs/kip-37.md

```diff
@@ -712,15 +712,27 @@ interface IKIP37Mintable {
         string calldata _uri
     ) external returns (bool);
 
-    /// @notice Mints tokens in a batch and assigns the tokens according to the variables `_to` and `_quantities`.
+    /// @notice Mints token and assigns the token according to the variables `_to` and `_quantity`.
     /// @dev Throws if `msg.sender` is not allowed to mint.
-    ///   MUST emit an event `TransferSingle` or `TransferBatch`.
+    ///   MUST emit an event `TransferSingle`.
     /// @param _id The token id to mint.
-    /// @param _to The list of addresses that will receive the minted tokens.
-    /// @param _quantities The list of quantities of tokens being minted.
+    /// @param _to The address that will receive the minted token.
```

Comment:
```suggestion
    /// @param _to The address that will receive the minted tokens.
```

Reply:
@kaikookim How do we define 'token'? A token is defined based on creation of mint?

Reply:
@kjhman21 'token' also includes 'FT', so it would be better to define it with all the tokens minted. I agree with changing 'token' to 'tokens'.